ServerCore/Pages/Responses/Create.cshtml.cs

```diff
@@ -20,28 +21,20 @@ public CreateModel(ServerCore.Models.PuzzleServerContext context)
 
         public int PuzzleId { get; set; }
 
-        public int EventId { get; set; }
-
         public IActionResult OnGet(int puzzleId, int eventId)
         {
             this.PuzzleId = puzzleId;
-            this.EventId = eventId;
             return Page();
         }
 
         public async Task<IActionResult> OnPostAsync(int puzzleId)
         {
-            if (!ModelState.IsValid)
```

Comment:
Other Create pages keep this validity check - why did you remove it here?

Reply:
We want to keep the validity checks in place - they  let us do things like add attributes to the properties in the data model and have those attributes be enforced all the way through the stack.